applications/launchpad/gui-react/src/components/NodeBox/index.tsx

```diff
@@ -18,14 +25,17 @@ import { NodeBoxContentPlaceholderProps, NodeBoxProps } from './types'
  * @param {CSSWithSpring} [style] - the box style
  * @param {CSSWithSpring} [titleStyle] - the title style
  * @param {CSSWithSpring} [contentStyle] - the content style
+ * @param {boolean} [helpPrompt] - renders help prompt button
```

Comment:
this does not correspond to the code added to props below

Reply:
Fixed.